oak/server/rust/oak_abi/src/label.rs

```diff
@@ -0,0 +1,234 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
```

Comment:
This didn't immediately look like it belonged here, because it doesn't give constant values or host function signatures for things that flow across the Wasm ABI boundary.  

However, I'm guessing that serialized `Label` protos will flow over the ABI boundary at some point in future, and then any Rust code on either side of the ABI boundary will have a use for these Rust helpers.

So maybe add a module doc comment explaining that?

Reply:
Done.